src/equations/numerical_fluxes.jl

```diff
@@ -37,6 +37,36 @@ end
 Base.show(io::IO, f::FluxPlusDissipation) = print(io, "FluxPlusDissipation(",  f.numerical_flux, ", ", f.dissipation, ")")
 
 
+"""
+    FluxRotated(numerical_flux)
+
+Compute a `numerical_flux` flux in direction of a normal vector by rotating the solution, 
+computing the numerical flux in x-direction, and rotating the calculated flux back.
+
+Requires a rotationally invariant equation with equation specific functions `rotate_to_x` and `rotate_from_x`.
+"""
+struct FluxRotated{NumericalFlux}
+  numerical_flux::NumericalFlux
+end
+
+@inline function (flux_rotated::FluxRotated)(u_ll, u_rr, normal_vector::AbstractVector, equations)
+  @unpack numerical_flux = flux_rotated
+
+  norm_ = norm(normal_vector)
+  # normalize(normal_vector) allocates
+  normal = SVector(normal_vector[1] / norm_, normal_vector[2] / norm_)
```

Comment:
We need the norm of the vector (that contains the metric terms) to scale the flux at the end.
For example, for a Cartesian mesh, the norm of the vector here would be just `dx/2`.
This factor is necessary, because we use a different Jacobian (`dx*dy/4`, the determinant of the Jacobian matrix of the mapping) than unstructured Trixi.

Reply:
Oh, right, I missed that. Thanks!